modules/apps/frontend-js/frontend-js-react-web/webpack.config.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+const path = require('path');
+const { container: { ModuleFederationPlugin } } = require('webpack');
+
+module.exports = {
+	context: path.resolve(__dirname),
+	devtool: 'source-map',
+	entry: './null.js',
```

Comment:
Let's move this part to the build tooling. I don't want to be creating a `null.js` file in every module

Reply:
Of course. That's one of the things we should make transparent and automatic as it is a bit horrible (but sadly, necessary).

Reply:
This has been moved to npm-scripts (see https://github.com/izaera/liferay-frontend-projects/commits/issue-250)